src/windows/about.py

```diff
@@ -160,12 +158,9 @@ def __init__(self):
             <html><head/>
             <body style="font-size:11pt; font-weight:400; font-style:normal;">
             <hr />
-            <p align="center"
-               style="margin:12px 12px 0 0; -qt-block-indent:0; text-indent:0;">
+            <p align="center" style="margin:12px 12px 0 0; -qt-block-indent:0; text-indent:0;">
                <span style="font-size:10pt; font-weight:600;">%s </span>
-               <a href="http://www.openshotstudios.com?r=about-us">
-               <span style="font-size:10pt; font-weight:600; text-decoration: none; color:#55aaff;">
-               OpenShot Studios, LLC<br /></span></a>
+               <a href="http://www.openshotstudios.com?r=about-us"><span style="font-size:10pt; font-weight:600; text-decoration: none; color:#55aaff;">OpenShot Studios, LLC<br /></span></a>
```

Comment:
176 characters (not including tabs) is a pretty long line. I sure wish I had a monitor that big :laughing:.

but I don't see the need to save lines if it might be less readable for edditing..

Reply:
Well we can consider this the first stage of improving the code.
There don't really need to be an span and an br tag inside the a tag and the URL redirects to a shorter URL and i suggested using "Website" as link test so i guess in the end it will be:

               <a href="https://www.openshot.org/">Website</a>

Reply:
the `<br />` is there so that the about window is evenly spaced. But you're right about the web link. Feel free to change that to `https://www.openshot.org/company` as that's the about current about page.

Reply:
Better use CSS margin/padding to control space between elements.